src/components/screens/bookmarks/bookmarksList/bookmarksList.js

```diff
@@ -92,13 +93,15 @@ class BookmarksList extends React.Component {
 
   onTitleChange({ target }) {
     this.setState({
-      eddittedTitle: target.value,
+      ...this.state,
```

Comment:
What's this for? setState merges the given object. You don't need to add this.

Reply:
true I forget remove it from the last change, is fixed now